(async () => {
  try {
    const text = "aaa";
    console.log(text);
  } catch (e) {
    console.error(e);
  }
})();